ansible/tasks/hashbang/templates/usr/local/share/man/man7/hashbang.7.j2

```diff
@@ -30,7 +30,7 @@ echo the foo file to the console
 
 .SH AVAILABLE SOFTWARE
 .SS Account Management
-hashbangctl - An account management program which can update your ssh keys, account name, and default shell.
+ssh hashbang.sh - interactive ssh application to manage keys, etc
```

Comment:
"manage keys, default shell, and user configuration"* so people know to not use chsh